.github/workflows/publish.yaml

```
@@ -30,6 +30,6 @@ jobs:
         run: make build
 
       - name: Create release
-        run: gh release create v$(uvx hatch version) dist/* --generate-notes --prerelease
+        run: gh release create v$(uvx hatch --with hatch-vcs version) dist/* --generate-notes --prerelease
```

```suggestion
        run: gh release create v$(uvx --with hatch-vcs hatch version) dist/* --generate-notes --prerelease
```

`--with hatch-vcs` is a parameter for uvx.